api_fhir_r4/views/fhir/subscription.py

```diff
@@ -2,19 +2,31 @@
 
 from api_fhir_r4.exceptions import FHIRException
 from api_fhir_r4.models import Subscription
+from api_fhir_r4.permissions import FHIRApiSubscriptionPermissions
 from api_fhir_r4.serializers import SubscriptionSerializer
 from api_fhir_r4.services import SubscriptionService
 from api_fhir_r4.views.fhir.base import BaseFHIRView
 
 
 class SubscriptionViewSet(BaseFHIRView, ModelViewSet):
+    _error_while_saving = 'Error while saving a subscription: %(msg)s'
```

Comment:
If you're using it in destroy, this message should be adjusted.

Reply:
I think that's a copy-paste error. I will fix that